test/parallel_api/algorithm/alg.merge/inplace_merge.pass.cpp

```diff
@@ -45,6 +45,31 @@ struct test_one_policy
         EXPECT_EQ_N(first1, first2, n, "wrong effect from inplace_merge with predicate");
     }
 
+    template <typename Policy, typename BiDirIt1, typename Size, typename Generator1, typename Generator2>
+    typename ::std::enable_if<!is_same_iterator_category<BiDirIt1, ::std::forward_iterator_tag>::value, void>::type
+    operator()(Policy&& exec, BiDirIt1 first1, BiDirIt1 last1, BiDirIt1 first2, BiDirIt1 last2, Size n, Size m,
```

Comment:
Could you please find a way to combine this one with another `operator()` to reduce code duplication?

Reply:
I did not find a way to do it because of problem with same mangled name.